PinpointKit/PinpointKit/Sources/SystemLogCollector.swift

```diff
@@ -6,12 +6,30 @@
 //  Copyright © 2016 Lickability. All rights reserved.
 //
 
+/**
+ The type of logs to collect.
+ 
+ - Application: Logs from the application target.
```

Comment:
It would probably be better (and more consistent with the rest of the project) to document the cases individually so that the docs will pop up when alt-clicking the case.